app/src/main/kotlin/com/exallium/todoapp/MainActivity.kt

```diff
@@ -46,7 +50,7 @@ class MainActivity : AppCompatActivity() {
         router = Conductor.attachRouter(this, container, savedInstanceState)
         router.addChangeListener(changeListener)
         if (!router.hasRootController()) {
-            router.setRoot(RouterTransaction.with(AllNotesViewImpl(Bundle())))
+            router.setRoot(RouterTransaction.with(AllNotesViewImpl(bundleFactory.createBundle())))
```

Comment:
This is good for consistency.  I'd say that since you're within the Android "layer" here, it'd be ok to just directly create a Bundle object, but this is fine.